terraform-modules/aws/amplify/main.tf

```diff
@@ -0,0 +1,60 @@
+resource "aws_iam_role" "amplify" {
+  name = "${var.name}-amplify-role"
+  tags = var.tags
+  assume_role_policy = jsonencode({
+    Version = "2012-10-17"
+    Statement = [
+      {
+        Action = "sts:AssumeRole"
+        Effect = "Allow"
+        Principal = {
+          Service = "amplify.amazonaws.com"
+        }
+      }
+    ]
+  })
+}
+
+#Base policy for Amplify app allows access to resources needed by Amplify applications.
+resource "aws_iam_role_policy_attachment" "role_attach" {
+  role       = aws_iam_role.amplify.name
+  policy_arn = "arn:aws:iam::aws:policy/AdministratorAccess-Amplify"
```

Comment:
Document this as the source of this policy for future reference: https://docs.aws.amazon.com/amplify/latest/userguide/security-iam-awsmanpol.html?authuser=1